### PR TITLE
New vfs baselines follow-up

### PR DESCRIPTION
**Pull request by rbuckton**
Per https://github.com/Microsoft/TypeScript/pull/20763#issuecomment-352553264, we opted to ["patch"](https://github.com/Microsoft/TypeScript/pull/20763#issuecomment-353210892) some of the paths used in the VFS PR to reduce the size of the PR and accept the baseline changes later. These are the baseline changes resulting from removing these "patches".

### CODE REVIEW

**Comment by mhegazy** [comment]
this will use `lib.d.ts` by default for tests, we opted to make the lib file smaller in tests to make sure we are not rechecking lib.d.ts all the time. what impact does this have on the test runtime?

File: tests/cases/conformance/expressions/operators/incrementAndDecrement.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
+// @lib: es5
```
Comment (weswigham) [inline tests/cases/conformance/expressions/operators/incrementAndDecrement.ts]: Why do so many tests now have an explicit `// @lib: es5`?

**Comment by rbuckton** [comment]
I ran `Measure-Command { gulp runtests-parallel }` against master and oddly enough this was faster:

```
C:\dev\TypeScript [master]> gulp clean
...
C:\dev\TypeScript [master]> gulp tests
...
C:\dev\TypeScript [master]> Measure-Command { gulp runtests-parallel }
...
TotalSeconds      : 310.9653838
...


C:\dev\TypeScript [vfsBaselines]> gulp clean
...
C:\dev\TypeScript [vfsBaselines]> gulp tests
...
C:\dev\TypeScript [vfsBaselines]> Measure-Command { gulp runtests-parallel }
...
TotalSeconds      : 300.0299655
...
```

**Comment by weswigham** [comment]
> oddly enough this was faster

AFAIK we don't _actually_ check the std libs we inject (unless added via the `libFiles` psuedo-compileroption) - we pull do whatever's necessary to check the test,. This PR removes a munch of errors from older baselines (by adjusting the lib they use); errors take time to generate, emit, and compare, so I'd wager that's where the difference lies.

**Pull request merged**